app/search/template.hbs

```diff
@@ -1,49 +1,99 @@
 {{!-- template-lint-disable no-bare-strings --}}
+{{page-title (t 'search.page-title')}}
 <OsfLayout 
     @backgroundClass='search-page'
 as |layout|>
     <layout.heading local-class='heading-wrapper'>
         <form role='search'>
-            <label local-class='heading-label'>
-                {{t 'search.search_header'}}
-            </label>
-            <Input
-                local-class='search-input'
-                @type='search'
-                @placeholder={{t 'search.textbox_placeholder'}}
-                @value={{this.q}}
-                {{on 'keydown' this.onKeyPress}}
-            />
-            <Button
-                local-class='search-button'
-                aria-label={{t 'search.search_button_label'}}
-                @type='primary'
-                @layout='large'
-                {{on 'click' (perform this.doSearch)}}
+            <label
+                for='search-input'
+                local-class='heading-label'
             >
-                <FaIcon @icon='search' />
-            </Button>
+                <h1>{{t 'search.search-header'}}</h1>
+            </label>
+            <span local-class='search-input-wrapper'>
+                <Input
+                    local-class='search-input'
+                    id='search-input'
+                    @type='search'
+                    @placeholder={{t 'search.textbox-placeholder'}}
+                    @value={{this.q}}
+                    {{on 'keydown' this.onKeyPress}}
+                />
+                <Button
+                    data-test-search-submit
+                    data-analytics-name='search button'
+                    local-class='search-button'
+                    aria-label={{t 'search.search-button-label'}}
+                    @type='primary'
+                    @layout='large'
+                    {{on 'click' this.doSearch}}
+                >
+                    <FaIcon @icon='search' />
+                </Button>
+            </span>
         </form>
 
-        {{#if this.showSidenavToggle}}
+        {{#if this.showSidePanelToggle}}
             <Button
-                aria-label={{t 'search.toggle_sidenav'}}
+                data-test-toggle-side-panel
+                data-analytics-name='toggle side panel'
+                aria-label={{t 'search.toggle-sidenav'}}
                 local-class='sidenav-toggle'
                 {{on 'click' layout.toggleSidenav}}
             >
                 <FaIcon @icon='bars' />
             </Button>
         {{/if}}
     </layout.heading>
-    <layout.left>
-        {{!-- current filters --}}
-        {{!-- filter facets --}}
-        Left
+    <layout.left
+        data-analytics-scope='search left panel'
+        local-class='left-panel'
+    >
+        {{!-- Object type filter dropdown if we are in mobile --}}
```

Comment:
You say, 'if', but I'm not seeing any logic here. Is there something I'm missing, or are you waiting until you actually put in the desktop tabs to do the mobile logic? Or is this OsfLayout magic?

Reply:
I'll be adding the object type filter and dropdown in a separate ticket, so this comment is more of a "`TODO` comment" more than an "information comment"